templates/ai/mlops.html

```diff
@@ -0,0 +1,210 @@
+{% extends "ai/_base_ai.html" %}
+
+{% block title %}Modular MLOps platform{% endblock %}
+
+{% block meta_description %}Modular MLOps platform that seamlessly integrates open source tooling to cover all stages of the Machine learning lifecycle.{% endblock %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/18AMX_-3zjBNUNGr0X5QtjKhsUzAc2k8VM1-opZEGh4k/edit{% endblock meta_copydoc %}
+
+{% block body_class %}is-paper{% endblock body_class %}
+
+{% block content %}
+
+<section class="p-strip--whitesuru is-shallow u-no-padding--bottom">
+  <div class="p-section--deep">
+    <div class="row">
+      <div class="col-start-large-4 col-9">
+        <div class="p-section--shallow">
+          <h1 class="u-no-padding--bottom">Deliver AI at scale <br class="u-hide--small">with secure and supported MLOps</h1>
+        </div>
+        <div class="p-section--shallow">
+          <p>Move from experimentation to production using a modular MLOps platform. <br class="u-hide--small">Accelerate your project delivery and ensure quick return on investment.</p>
+        </div>
+        <a class="p-button--positive" href="/engage/mlops-guide">Read our guide to MLOps</a>
+        <a class="p-button js-invoke-modal" href="#get-in-touch">Talk to our experts</a>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="row">
+    <hr class="p-rule">
+    <div class="col-start-large-4 col-9">
+      <div class="p-section--shallow">
+        <h2>What is MLOps?</h2>
+      </div>
+      <p>Machine learning operations (MLOps) is like DevOps for machine learning. It is a set of practices that automates machine learning workflows, ensuring scalability, portability and reproducibility.</p>
+      <div>
+        <img src="https://assets.ubuntu.com/v1/0f2554e7-what is MLOps illustration.svg" alt="" style="height: 267px; width: 597px;">
```

Comment:
Was this the image you spoke about being converted to a PNG when using the image template?

Reply:
Yes, Lyubo noticed a difference between the version of the image straight from the asset server and how it appears after it's been sanitized by cloudinary so we agreed to just link the asset directly